gaia/parser.py

```diff
@@ -33,6 +36,9 @@
 # TODO: Need a method to register classes from domain specific code
 valid_classes.extend([x[0] for x in inspect.getmembers(
     gaia.geo.geo_inputs, inspect.isclass) if x[0].endswith('IO')])
+for plugin in get_plugins():
+    valid_classes.extend([x[0] for x in inspect.getmembers(
```

Comment:
@mbertrand Rather than add all classes from a plugin,  can we add something to the plugin API like ```PLUGIN_CLASS_EXPORTS``` which must be present on the plugin and explicitly defines what classes should be included?